criu/namespaces.c

```diff
@@ -282,6 +282,46 @@ int restore_ns(int rst, struct ns_desc *nd)
 	return ret;
 }
 
+int switch_mnt_ns(int pid, int *rst, int *cwd_fd)
+{
+	int ret;
+	int fd;
+
+	if (!cwd_fd)
+		return switch_ns(pid, &mnt_ns_desc, rst);
+
+	fd = open(".", O_DIRECTORY);
+	if (fd < 0)
```

Comment:
Need to print an error here in other places, otherwise it will be unclear what syscall returns an error.